sdk/azidentity/README.md

```diff
@@ -0,0 +1,279 @@
+# Azure Identity client library for Go
+
+The Azure Identity library provides a set of credential classes for use with
```

Comment:
I wouldn't call them classes since Go doesn't have classes.  Maybe "types"?

Reply:
Yes agreed, just made a fuller search to remove all references to classes

Reply:
Done